doc/Language/nativecall.pod6

```diff
@@ -185,6 +185,19 @@ When the signature of your native function needs a pointer to some native type
     # C prototype is void my_version(int *major, int *minor)
     sub my_version(int32 is rw, int32 is rw) is native('foo') { * }
     my_version(my int32 $major, my int32 $minor); # Pass a pointer to
+    
+When you just want to use/pass a pointer using Pointer class you have to 
+create the pointers when you declare them. Then you can do this without specifying 
+what type it points to. Here is the exmaple:
+    use NativeCall;
+    # C prototype is void create_object(void **object)
+    sub create_object(Pointer is rw) is native('foo') { * }
+    my Pointer $p = Pointer.new();
+    create_object($p); # pointer is set by create object
+    
+Pointer class can be used with types as well e.g. Pointer<Str>..
```

Comment:
^ The
Again, please use `C<>`

Reply:
Are you talking to_

Reply:
Er... It's a comment on the line pointed to above. Basically, it says "introduce "The" before "Pointer"" and use pod6 marks for code (`C<>`) Who was I going to talk to?